sql-parser/src/main/java/io/crate/sql/tree/Insert.java

```diff
@@ -59,6 +60,12 @@ public DuplicateKeyType getDuplicateKeyType() {
         return duplicateKeyType;
     }
 
+    /**
+     *  If null, no duplicate key behavior has been specified.
+     *  Otherwise, perform the updates in the list.
+     *  List may be empty in case nothing is done on duplicates.
+     */
+    @Nullable
     public List<Assignment> onDuplicateKeyAssignments() {
```

Comment:
I wonder if we should be a bit more explicit about this and introduce some kind of `OnConflict` structure ?
Not sure if it's worth it, but in other places we handle nullable lists interchangeable with empty lists and this might be a bit confusing.

Reply:
I thought it would be nice to encode this into the existing behavior but I realized the behavior might not be obvious unless documented. Your comments confirms this suspicion :)